starknet_py/tests/e2e/client/gateway_test.py

```diff
@@ -38,6 +39,24 @@ async def test_get_class_by_hash_sierra_program():
     assert isinstance(contract_class.abi[0], dict)
 
 
+@pytest.mark.asyncio
+async def test_get_new_state_update():
+    client = GatewayClient(
+        net="https://external.integration.starknet.io"
+    )  # TODO: Replace this with fixture
+
+    state_update = await client.get_state_update(block_number=283364)
+
+    assert state_update.state_diff.replaced_classes == []
```

Comment:
Would be awesome to find a block with something in `replaced_classes`

Reply:
Tried replacing class hashes myself and such block now exists `0x656164aefe7d646927fcbf8c576420475fcd5602bf284c00aa00eb3d1b8151b`.